### PR TITLE
FORMS-10107 Fix id mismatch issue with embed form having fragment

### PR DESCRIPTION
**Pull request by dmaurya929**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractComponentImpl.java [hunk 1]
```diff
@@ -129,7 +129,8 @@ private void init() {
         if (currentPage != null && resource != null && request.getAttribute(REQ_ATTR_EMBEDDED_ADAPTIVEFORM) != null) {
             if (!GuideWCMUtils.isForms(getCurrentPage().getPath())) {
                 PageManager pageManager = currentPage.getPageManager();
-                Page resourcePage = pageManager.getContainingPage(resource);
+                String pagePath = (String) request.getAttribute(REQ_ATTR_EMBEDDED_ADAPTIVEFORM);
```
Comment (rismehta) [inline bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractComponentImpl.java:132]: What if is this attribute is not set ?

File: it/content/src/main/content/META-INF/vault/filter.xml [hunk 2]
```diff
@@ -8,4 +8,5 @@
     <filter root="/conf/core-components-it" mode="update"/>
     <filter root="/conf/wknd" mode="update"/>
     <filter root="/conf/datalayerContainer" mode="update"/>
+    <filter root="/content/core-components-examples/aemembedcontainertest"/>
```
Comment (rismehta) [inline it/content/src/main/content/META-INF/vault/filter.xml:11]: All sites specific use-case should be present under `/content/forms/sites`. No changes in existing filter is allowed unless there are new use-cases

**Review by rismehta — CHANGES_REQUESTED** [review]
-1 for now.

File: bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractComponentImpl.java (hunk 1 above)
Comment (dmaurya929) [reply bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractComponentImpl.java:132]: In line 129 there is already a condition in place to check if this attribute is not null. Cypress test covers this scenario, let me see about the unit test.

**Pull request merged**